app/src/test/java/mozilla/lockbox/store/ClipboardStoreTest.kt

```diff
@@ -58,4 +58,19 @@ class ClipboardStoreTest : DisposingTest() {
         val clipboardManager: ClipboardManager = RuntimeEnvironment.application.getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
         assertTrue(clipboardManager.primaryClip.getItemAt(0).text.equals(testString))
     }
+
+    @Test
+    fun testReplaceDirty() {
+        val testString = "my_test_password"
+        val dirtyString = ""
```

Comment:
you can say that dirty is the one that will be erased however you can say that dirty is the one that replace the clean one with what you want in order to prevent that someone will still that password

Depend on what angle you want to see the problem.

Reply:
See comment on `ClipboardStore`.  This phrasing if very confusing, which is going to lead to bugs in the future.